docs/source/design/control_plane.rst

```diff
@@ -4,18 +4,17 @@
 FlyteRemote: A Programmatic Control Plane Interface
 ###################################################
 
-For those who require programmatic access to the control plane, :mod:`~flytekit.remote` module enables you to perform
+For those who require programmatic access to the control plane, the :mod:`~flytekit.remote` module enables you to perform
 certain operations in a python runtime environment.
 
 Since this section naturally deals with the control plane, this discussion is only relevant for those who have a Flyte
-backend set up, and have access to it (a local sandbox will suffice as well of course). These objects do not rely on the
-underlying code they represent being locally available.
+backend set up and have access to it (a :std:ref:`local sandbox <flyte:deployment-sandbox>` will suffice as well).
 
 ***************************
 Create a FlyteRemote Object
 ***************************
 
-The :class:`~flytekit.remote.FlyteRemote` class is the entrypoint for programmatically performing operations in a python
+The :class:`~flytekit.remote.remote.FlyteRemote` class is the entrypoint for programmatically performing operations in a python
```

Comment:
nit: why the double remote in the path? isn't it exposed in flytekit/remote/__init__.py ?

Reply:
yes, but sphinx needs the fully qualified path for the link to show up